ManageWikiNamespaces.php

```diff
@@ -343,4 +343,15 @@
 		'help' => '',
 		'requires' => [],
 	],
+	'wgModerationOnlyInNamespaces' => [
+		'name' => 'Allow non-automoderated users to bypass moderation only in this namespace?',
```

Comment:
That's not what it does. If `$wgModerationOnlyInNamespaces` is not `[]` (default), then moderation is disabled in all namespaces except those listed in `$wgModerationOnlyInNamespaces`.